File: storage/data_transpiled/at_coder/agc011/B/4669498.java
static final int [ ] getLevels ( ) {
  final int INF = 10 * * 18 ;
  final int MOD = 10 * * 9 + 7 ;
  {
    final int [ ] LI = new int [ N ] ;
    for ( int i = 0 ;
    i < N ;
    i ++ ) {
      LI [ i ] = Integer . parseInt ( System . console ( ) . readLine ( ) ) ;
    }
  }
  {
    final int [ ] LI_ = new int [ N ] ;
    for ( int i = 0 ;
    i < LI_ . length ;
    i ++ ) {
      LI_ [ i ] = Integer . parseInt ( System . console ( ) . readLine ( ) ) - 1 ;
    }
  }
  {
    final int [ ] LF = new int [ N ] ;
    for ( int i = 0 ;
    i < LF . length ;
    i ++ ) {
      LF [ i ] = Float . parseFloat ( System . console ( ) . readLine ( ) ) ;
    }
  }
  {
    final int [ ] LS = new int [ N ] ;
    for ( int i = LF . length ;
    i < LS . length ;
    i ++ ) {
      LS [ i ] = LF [ i ] ;
    }
  }
  {
    final int N = II [ N ] ;
    for ( int i = 0 ;
    i < S . length ;
    i ++ ) {
      N = S [ i ] ;
    }
  }
}

File: storage/data_original/at_coder/agc011/B/4669498.py
import sys
sys.setrecursionlimit(10**7)
INF = 10 ** 18
MOD = 10 ** 9 + 7
def LI(): return [int(x) for x in sys.stdin.readline().split()]
def LI_(): return [int(x) - 1 for x in sys.stdin.readline().split()]
def LF(): return [float(x) for x in sys.stdin.readline().split()]
def LS(): return sys.stdin.readline().split()
def II(): return int(sys.stdin.readline())
def SI(): return input()

from itertools import accumulate

def main():
    N = II()
    A = LI()
    A.sort()
    S = list(accumulate(A))
    ans = 1
    for s, a in zip(S[-2::-1], A[-1::-1]):
        if s * 2 < a:
            break
        ans += 1

    return ans

print(main())